imports/ui/Body.js

```diff
@@ -15,26 +14,31 @@ import './Index.js';
 import './HongKongNewGame.js';
 import './JapaneseNewGame.js';
 
-Template.JapaneseRanking.helpers({
-	japanesePlayers() {
-		return Players.find({}, {sort: { japaneseElo: -1}});
-	},
+var index;
+
+function initPlayer(key) {
+	index = 1;
+
+	var sort = {};
+	sort[key] = -1;
```

Comment:
We should be able to specify other sort directions if, say, we want to sort by chombo# ascending.